### DISCUSSION TITLE
피그마 디자인 시스템에 디자인폰트와 숫자에만 적용하는 폰트가 추가되었습니다!

### DISCUSSION THREAD

**Discussion by yoonucho** [OttSideProject/OnDe-FE#7] [Ideas]
안녕하세요! 폰트 추가 관련으로  몇 가지 제안을 드리려고 합니다.

우선, 디자인 폰트 관련해서 말씀드리면, 메인 홈에서만 사용되는 폰트는 이미 설정해두었는데, 숫자에만 사용할 전용 폰트도 작업을 진행하신 분이 없다면 제가 설정해두려고 합니다.

![화면 캡처 2024-09-06 011800](https://github.com/user-attachments/assets/0a72e0fe-da3a-4cda-bb88-2c8e58787f61)

또한, 기존에 설정된 Pretendard 폰트는Regular와 SemiBold가 함께 사용되고 있었는데, 총 4개의 폰트를 유지하는 것보다 SemiBold는 지우고 Regular 하나로 통일한 뒤, font-weight를 통해 폰트 굵기를 조정하는 방식이 더 효율적일 것 같다는 생각이 듭니다. 

마지막으로, Pretendard 폰트를 더 경량화된 서브셋 폰트(subset font)로 교체하려고 합니다. 서브셋 폰트는 불필요한 글자를 제거하고 실제로 사용할 글자만 남겨둔 폰트인데, 이를 적용하면 성능 최적화에 도움이 될 것 같습니다.

이렇게 진행해도 괜찮을지 의견 부탁드립니다. 혹시 더 나은 방안이 있다면 자유롭게 알려주세요!

감사합니다 :)

**Comment by keemewoon** [comment]
안녕하세요! 내용 확인했습니다 서브셋폰트라는것이 있는줄은 처음 알았네요. 요렇게 진행해주심 좋을것 같습니다👍
